test/DynamoCoreWpf3Tests/NodeAutoCompleteSearchTests.cs

```diff
@@ -502,8 +502,8 @@ public void NodeSuggestions_DefaultSuggestions()
 
             // The initial list will fill the FilteredResults with a few options - all basic input types
             searchViewModel.PopulateAutoCompleteCandidates();
-            Assert.AreEqual(5, searchViewModel.FilteredResults.Count());
-            Assert.AreEqual("String", searchViewModel.FilteredResults.FirstOrDefault().Name);
+            Assert.AreEqual(3, searchViewModel.FilteredResults.Count());
+            Assert.AreEqual("List Create", searchViewModel.FilteredResults.FirstOrDefault().Name);
```

Comment:
The type `var[]..[]` means a type of arbitrary rank, which means it could also accept a single value (rank 0). In such a case, I don't believe prompting for a list input to be connected to this port is always the most appropriate option.

Reply:
I think it's a valid option/close enough, previous matching was more erroneous imo. Also the input in the node in the graph used in the test is actually named list.